app/src/main/java/ar/com/wolox/android/example/ui/home/MainActivity.java

```diff
@@ -0,0 +1,20 @@
+package ar.com.wolox.android.example.ui.home;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+
+/**
+ * Application home;
+ */
+public class MainActivity extends WolmoActivity {
```

Comment:
Maybe we can change the name to something like HomeActivity. What do you think?

Reply:
I think is better HomeActivity because the main (entry) activity for the App is LoginActivity